extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -117,6 +119,36 @@ export class AmpAdUIHandler {
   }
 
   /**
+   * See BaseElement method.
+   */
+  createPlaceholderCallback() {
+    if (!isExperimentOn(this.baseInstance_.win, UX_EXPERIMENT)) {
```

Comment:
Yup that experiment name is `amp-ad-loading-ux`. it makes sense to me to put fallback + placeholder at same place.